Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/DataDonationModule.kt

```diff
@@ -10,6 +10,7 @@ import de.rki.coronawarnapp.datadonation.survey.SurveyModule
 @Module(
     includes = [
         SurveyModule::class,
+        SurveyModule.ResetModule::class,
```

Comment:
it is already included in `ResetTestComponent`, why again?

Reply:
Removed it, because its included in the SurveyModule, which is included here already.